js/ui/drop_down_button.d.ts

```diff
@@ -306,11 +305,16 @@ export interface dxDropDownButtonItem extends dxListItem {
     /**
      * @docid
      * @default null
-     * @type_function_param1_field3 model:object
+     * @type_function_param1 e:object
+     * @type_function_param1_field1 component:dxDropDownButton
+     * @type_function_param1_field2 element:DxElement
+     * @type_function_param1_field3 model:any
      * @type_function_param1_field4 event:event
+     * @type_function_param1_field5 itemData:object
+     * @type_function_param1_field6 itemElement:DxElement
      * @public
      */
-    onClick?: ((e: { component?: dxDropDownButton; element?: DxElement; model?: any; event?: DxEvent }) => void) | string;
+     onClick?: ((e: ItemClickEvent) => void) | string;
```

Comment:
Does runtime support this? I mean extra fields: `itemData` and `itemElement`

Reply:
Sure